registration/registration-client/src/main/java/io/mosip/registration/controller/reg/DemographicDetailController.java

```diff
@@ -1757,6 +1757,16 @@ private void setTextFieldListener(TextField textField) {
 					LOGGER.debug(loggerClassName, APPLICATION_NAME, RegistrationConstants.APPLICATION_ID,
 							"Setting secondary langauge text");
 
+					// excempted column list
+					String[] exemptedTransliteration = new String[]{"email", "nationalIdentityNumber", "phoneNumber"};
+					List<String> exemptList = new ArrayList<>(Arrays.asList(exemptedTransliteration));

```

Comment:
Arrays.asList(exemptedTransliteration) itself returns List<String> so this statement can be changed to 

List<String> exemptList = Arrays.asList(exemptedTransliteration);

Reply:
Updated the PR, retrieved from properties, also fixed a issue in ClientSettingSyncHelper, where master sync fails if there are dynamic columns with same name for different languages